thorlcr/master/thmastermain.cpp

```diff
@@ -954,6 +956,18 @@ int main( int argc, const char *argv[]  )
                 startPerformanceMonitor(pinterval, PerfMonStandard, nullptr);
 
             // NB: workunit/graphName only set in one-shot mode (if isCloud())
+
+            if (workunit)
+            {
+                Owned<IWorkUnitFactory> factory = getWorkUnitFactory();
+                Owned<IConstWorkUnit> wuRead = factory->openWorkUnit(workunit);
+                if (wuRead)
+                {
+                    Owned<IWorkUnit> wuWrite = &wuRead->lock();
+                    updateWorkunitStat(wuWrite, SSTglobal, nullptr, StTimeThorStartup, nullptr, thorStartupTimer.elapsedNs());
```

Comment:
I think this should be updating the startup time for the graph, rather than global.